### ISSUE TITLE
Long first startup

### ISSUE THREAD

**Issue by MrQubo** [Popax21/synaTudor#28]
`tudor-host-launcher.service` takes some time to start. I think it would be better to start it automatically on system startup instead of on the first use via dbus.

**Comment by Popax21** [comment]
> `tudor-host-launcher.service` takes some time to start. I think it would be better to start it automatically on system startup instead of on the first use via dbus.

Just starting `tudor-host-launcher` won't do much - you want to auto-launch `fprintd` instead, which will cause the host launcher to start as well. To do so, you can simply use a systemd overlay file to add it to a boot target of your choosing; I do this as well on my own laptop.

**Comment by MrQubo** [comment]
Maybe we should install this overlay by default to add `fprintd` to `multi-user.target`?

**Comment by Popax21** [comment]
> Maybe we should install this overlay by default to add `fprintd` to `multi-user.target`?

I prefer not interfering with the configuration files of other services, so I wouldn't be in favor of that. Instead, I think documenting this somewhere so that interested users can make the modification themselves is the better way forward here.

**Comment by major-mayer** [comment]
Could you briefly explain what kind of files are needed to reduce the startup time of fprintd?
I don't understand what it means to ...
> simply use a systemd overlay file to add it to a boot target of your choosing

**Comment by Popax21** [comment]
> Could you briefly explain what kind of files are needed to reduce the startup time of fprintd? I don't understand what it means to ...
> 
> > simply use a systemd overlay file to add it to a boot target of your choosing

TL;DR: execute `systemctl edit fprintd` as root, then paste in the following contents:
```
[Install]
WantedBy=multi-user.target
````

This will create a systemd drop-in file (drop-in files can be used to override systemd unit file settings) which adds fprintd to `multi-user.target` - this causes fprintd to automatically start on boot.

**Comment by major-mayer** [comment]
Great thanks for the quick help.
I guess you meant "systemctl edit fprintd", but then it works flawlessly.
